engine/src/main/java/com/redhat/cloud/notifications/db/repositories/EndpointRepository.java

```diff
@@ -125,10 +128,11 @@ public List<Endpoint> getTargetEmailSubscriptionEndpoints(String orgId, UUID eve
      * Increments the server errors counter of the endpoint identified by the given ID.
      * @param endpointId the endpoint ID
      * @param maxServerErrors the maximum server errors allowed from the configuration
+     * @param minDelaySinceFirstServerErrorBeforeDisabling the minimum delay since the first error before an endpoint could be disabled
      * @return {@code true} if the endpoint was disabled by this method, {@code false} otherwise
      */
     @Transactional
-    public boolean incrementEndpointServerErrors(UUID endpointId, int maxServerErrors, int currentServerErrors) {
+    public boolean incrementEndpointServerErrors(UUID endpointId, int maxServerErrors, int currentServerErrors, Duration minDelaySinceFirstServerErrorBeforeDisabling) {
```

Comment:
`EndpointRepository` could inject the config value directly instead of passing it as a method arg here.